java/connector-node/risingwave-connector-test/src/test/java/com/risingwave/connector/source/SourceTestClient.java

```diff
@@ -41,7 +41,7 @@ public class SourceTestClient {
     static final Logger LOG = LoggerFactory.getLogger(SourceTestClient.class.getName());
 
     // default port for connector service
-    static final int DEFAULT_PORT = 50051;
+    static final int DEFAULT_PORT = 60051;
```

Comment:
No compatible issue?

Reply:
No. This port is only used in the unit test in Java, it will start the connector node on this port. We don't use it in public locations.